### PR TITLE
Gitignore sqlite related files in the db folder

### PR DESCRIPTION
**Pull request by wwahammy**
Occasionally, when a spec run is terminated in the middle, there's a `spec/internal/db/devise-authy.sqlite-shm` file. I've also seen one with another file extension starting with `sqlite-` (I don't remember the specific one). These are really just internal to how SQLite handles journaling and error handling. Since those SQLite dbs are only used as part of the spec run, we don't want people committing these files either.

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -1,27 +1,22 @@
...
-        ruby: [2.5, 2.6, 2.7, "3.0", 3.1, head]
-        gemfile: [rails_5_2, rails_6]
-        exclude:
-          - ruby: "3.0"
-            gemfile: rails_5_2
-          - ruby: 3.1
-            gemfile: rails_5_2
-          - ruby: head
-            gemfile: rails_5_2
-    continue-on-error: ${{ endsWith(matrix.ruby, 'head') }}
```
Comment (BintLopez) [inline .github/workflows/build.yml:20]: Are these changes related to ignoring sqlite?

Comment (wwahammy) [reply .github/workflows/build.yml:20]: Hmmm, they're not. I'll clean it up quick.

**Review by camckin10 — APPROVED** [review]
✅
